app/assets/javascripts/application.js

```diff
@@ -1,9 +1,3 @@
-// This is a manifest file that'll be compiled into including all the files listed below.
-// Add new JavaScript/Coffee code in separate files in this directory and they'll automatically
-// be included in the compiled file accessible from http://example.com/assets/application.js
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// the compiled file.
-//
```

Comment:
Why you have removed this?

Reply:
because I don't think that's very useful, I can rewrite the comment in a separate PR if you prefer

Reply:
I added a better comment